src/TTCache.php

```diff
@@ -217,4 +224,42 @@ public function clearTags(string ...$tags) : void
         $htags = array_map([$this, 'hashedTag'], $tags);
         $this->cache->clearTags(...$htags);
     }
+
+    /**
+     * Returns an instance of TTCache specifically configured for use w/
+     * Cache\Adapter\Common\AbstractCachePool instances. It prevents the use of
+     * the use of reserved characters in cache keys
+     *
+     * @param AbstractCachePool $cachePool
+     * @return TTCache
+     */
+    public static function newWithCachePool(AbstractCachePool $cachePool): TTCache
```

Comment:
Gonna say Nope to this and the other methods below. Would prefer not to have each implementation go through its own separate constructor and us maintaining a repository of `N` constructors.